gdb-xtensa-esp32-elf/build.sh

```diff
@@ -1,42 +1,19 @@
 #!/bin/bash
```

Comment:
Nit: I would prefer we stick to pretty much the same script we use in the arm-none-eabi script just for consistency.

```
#!/bin/bash

set -ex

export TARGET=xtensa-esp32-elf
export TARGET_PREFIX="${PREFIX}/${TARGET}"

mkdir -p "${PREFIX}"/bin
mkdir -p "${TARGET_PREFIX}"

# Only copy what is necessary or else we'll copy a bunch of internal Conda stuff.
cp -R $SRC_DIR/bin     $TARGET_PREFIX
cp -R $SRC_DIR/include $TARGET_PREFIX
cp -R $SRC_DIR/lib     $TARGET_PREFIX
cp -R $SRC_DIR/share   $TARGET_PREFIX

# Symlink every binary from the build into /bin
pushd "${PREFIX}"/bin
    ln -s ../"${TARGET}"/bin/* ./
popd
```

The reason I removed the * was because we were copying `build_env_setup.sh`, `conda_build.sh`, and `metadata_conda_debug.yaml` into the ending recipe under `xtensa-esp32-elf` folder.

Reply:
heh i never knew `-R`
```
       -R, -r, --recursive
```

Reply:
but yeah makes sense, this was just the hacky first attempt, moved on to testing the functionality instead of cleaning up. will fix.

Reply:
btw- i'm much in favor of subshell `()` vs `pushd/popd`, much less error prone imo

Reply:
> btw- i'm much in favor of subshell () vs pushd/popd, much less error prone imo

I think my comment was over-bearing and probably not the right one to make. Sorry about that. Happy to have both of the places be updated - I just hadn't seen that before. Either way, I'm usually in favor of keeping the same approach or changing them all.